en/docs/develop/customizations/customizing-the-developer-portal/overriding-developer-portal-theme.md

```diff
@@ -1,8 +1,13 @@
 # Overriding Developer Portal Theme
 
-Developers can easily customize the Developer Portal UI by modifying a single JSON file which holds the parameterized constraints of the look and feel. For example, you can change the font family for the entire Developer Portal by modifying the JSON file.  Hence, developers are not required to have any prior knowledge on React, CSS, or HTML to customize the UI.
-The Developer Portal theme can be customized not only for the look and feel but also for changing the listing view from default to grid view or vice versa, hiding social features, etc.
+There are mainly 2 ways to customize and override the developer portal theme. 
 
+1. Theming using defaultTheme.js
+2. Overriding React components
+
+In this section, we will cover the first technique which is the easiest way to change the appearance of Developer portal.
```

Comment:
Shall we use the word `method` instead of the word `technique` 😃  just a suggestion @shaniR WDYT ?

```suggestion
In this section, we will cover the first method which is the easiest way to change the appearance of Developer portal.
```